docs/data-management/definitions/assignment-sql.md

```diff
@@ -20,75 +22,156 @@ SELECT
+Secondary IDs can help here. In addition to adding `anonymous_id` as the subject key ID, you can add `user_id` as a secondary ID:
 
-<!-- <img src="https://firebasestorage.googleapis.com/v0/b/eppo-documentation-images.appspot.com/o/add-assignment-sql-dimensions.png?alt=media&token=dfd583db-4ea7-4013-b5fc-d90612118738" width="500" height="200"/> -->
+| Column | Type in Eppo |
+|--------|--------------|
+| `assignment_timestamp` | Timestamp |
+| `anonymous_id` | Experiment subject ID |
+| `experiment` | Experiment key |
+| `variant` | Variant |
+| `user_id` | Secondary ID |
 
-![Add Assignment SQL Properties](/img/building-experiments/add-assignment-sql-dimensions.png)
+Any experiment analysis that uses this Assignment SQL Definition will have both anonymous ID and user-level metrics available. User metrics will be attributed back to the first associated anonymous ID in the assignment table. For more details, see the page on  [analyzing anonymous user experiments](/guides/advanced-experimentation/anonymous-explainer/).
 
-Holdout columns are also defined here. More information is available on the [dedicated page](/experiment-analysis/holdouts).
+### Email marketing experiments
 
-![Write Assignment SQL Query](/img/experiments/holdouts/holdouts-assignment-sql.png)
+When analyzing email marketing experiments, it's common to measure both overall user level metrics (engagement, revenue, retention, etc.) and campaign-specific metrics (click through rate, unsubscribes, etc.). In this case you may want to join only on `user_id` in some cases, and `email_send_id` in other cases. This is also supported by secondary IDs:
 
-8. Click **Save & Close**
+| Column | Type in Eppo |
+|--------|--------------|
+| `assignment_timestamp` | Timestamp |
+| `user_id` | Experiment subject ID |
+| `experiment` | Experiment key |
+| `variant` | Variant |
+| `email_send_id` | Secondary ID |
 
-## Handling duplicate assignments
+Similar to the pre-authentication use case described above, experiment analyses that use this assignment source can measure both user-level metrics and campaign specific metrics.
 
-Eppo has built in filtering to handle scenarios where there are multiple assignment events for a subject with different data. Here are a few common scenariios that you may encounter and how Eppo handles them:
 
-**1. Subject has multiple assignment events where they received multiple variations.**
+## Assignment deduplication
 
-Example: A subject received both the `Control` and `Treatment` within an experiment. 
+Eppo will automatically deduplicate assignment logs from the same subject-experiment pair by only considering the first record. Eppo will also gracefully handle scenarios where assignment data varies over time for the same subject. Details are described below.
```

Comment:
Might be good to clarify that it's the first record that occurs during the experiment's date window. I often get questions like "If we set the assignment start date as November 1st, but there are a few assignments in the table for that exp from October 30th, which one does Eppo consider 'first'?"